resources/logfs/dl.css

```diff
@@ -0,0 +1,42 @@
+/* Styles match microbit.org */
+body {
+  font-family: "Helvetica Now", Helvetica, Arial, sans-serif;
+}
+button {
+  border-radius: 23px;
+  min-height: 46px;
+  font-weight: 700;
+  font-size: 15px;
+  letter-spacing: 0.3px;
+  padding: 0 16px;
+}
+/* Primary button */
+button.p {
+  border: none;
+  background: #000;
+  color: #fff;
+}
+/* Secondary button */
+button.s {
+  color: #000;
+  border: 2px solid #000;
+  background: #fff;
+}
+tr:hover {
+  background-color: #ddd;
+}
+header,
+footer {
+  /* We can't afford the bytes to have a wrapper element and better CSS in the main doc
+     so use -ve margin to get a full width header. */
```

Comment:
This makes no sense now I've simplified the header styling.

Remove assuming we don't reinstate a black strip header after discussions today.

Reply:
We've also got a wrapper element now so can be tidied either way.